R/Rules-class.R

```diff
@@ -925,6 +925,65 @@ StoppingMTDdistribution <- function(target,
                              prob=prob)
 }
 
+#' Stop based on precision of MTD calculated as CV(MTD)
+#' 
+#' @description `r lifecycle::badge("experimental")`
+#' 
+#'
+#' @slot target (`numeric`)\cr toxicity target of MTD
+#' @slot threshCV (`numeric`)\cr threshold for CV to be considered accurate enough
+#'  to stop the trial
```

Comment:
```suggestion
#' @slot target (`numeric`)\cr toxicity target of MTD.
#' @slot threshCV (`numeric`)\cr threshold for CV to be considered accurate enough
#'  to stop the trial.
```

Reply:
Thanks for the suggestions and comments. I made all changes to the code (with one exception, please see comment below).
I think we should go over your comments and suggestion during the next meeting as I think it will be helpful for anyone.
I will start work on the test and _pkgdown.yaml, but I am still unsure how to do it. Probably something to look at in the next meeting.

Reply:
OK cool, agree, we can do that in the upcoming meeting!